code/modules/atmospherics/machinery/atmosmachinery.dm

```diff
@@ -12,33 +12,45 @@
 
 /obj/machinery/atmospherics
 	anchored = TRUE
-	move_resist = INFINITY				//Moving a connected machine without actually doing the normal (dis)connection things will probably cause a LOT of issues.
+	move_resist = INFINITY				//Moving a connected machine without actually doing the normal (dis)connection things will probably cause a LOT of issues. (this imply moving machines with something that can push turfs like a megafauna)
 	idle_power_usage = 0
 	active_power_usage = 0
 	power_channel = AREA_USAGE_ENVIRON
 	layer = GAS_PIPE_HIDDEN_LAYER //under wires
 	resistance_flags = FIRE_PROOF
 	max_integrity = 200
 	obj_flags = CAN_BE_HIT | ON_BLUEPRINTS
+	///Check if the object can be unwrenched
 	var/can_unwrench = 0
+	///Check what are the initialized directions
```

Comment:
```suggestion
	///Should we check the initialized directions
```
I assume this is a true/false thing? if you want brownie points fix the 0s in place of falses and 1s in place of trues

Reply:
No is not true/false, i have to change it since i think it stores bitflag with the directions (NORTH | SOUTH | EAST | WEST)

Yes after checking this is a bitflag, so i'll correct that, also the var just above this (`can_unwrench`) IS a true/false, so i'll fix that